ykman-gui/qml/DeviceInfo.qml

```diff
@@ -87,6 +87,12 @@ Item {
                                                                            "Not available"))
                     }
                 }
+                Button {
+                    Layout.alignment: Qt.AlignRight
+                    text: qsTr("Configure")
+                    enabled: device.enabled.indexOf('OTP') >= 0
```

Comment:
This should probably use the `isEnabled` function, no?

Reply:
Absolutely, updated PR.